rollup.config.ts

```diff
@@ -90,7 +90,7 @@ export default function rollup(options: RollupOptions): RollupOptions[] {
       name: 'react-query-devtools-noop',
       packageDir: 'packages/react-query-devtools',
       jsName: 'ReactQueryDevtools',
-      outputFile: 'react-query-devtools',
+      outputFile: 'noop',
```

Comment:
Setting this to `noop` will allow `react-query-devtools` and `react-query-devtools-noop` to generate their own output files